src/rust/ensogl/lib/core/src/display/scene.rs

```diff
@@ -311,6 +311,8 @@ impl Mouse {
         let frp             = frp::io::Mouse::new();
         let on_move         = mouse_manager.on_move.add(f!([frp,scene_frp,position,last_position]
             (event:&mouse::OnMove) {
+                let as_event:&web_sys::Event = event.as_ref();
+                scene_frp.processed_event_source.emit(Some(as_event.clone()));
```

Comment:
this line and the line with None need very detailed explanation. I understand it becaus of our in depth call recently, but in few weeks it will be a complete mystery for me. Maybe refactor them to helper function and create a doc for it desciribng what it really does in detail (or pointing to a place that describes it) ?